site/documentation/index.md

```diff
@@ -69,3 +70,29 @@ title: Apache Metron (Incubating) Documentation
         <a class="button-default" href=" https://cwiki.apache.org/confluence/display/METRON/Documentation" target="_blank">LEARN MORE</a>
     </div>
 </section>
+
+<section class="events-section info-block darken" id="releases">
+    <div class="wrapper downloads">
+        <div class="text-center">
+            <h4>Current Release</h4>
+        </div>
+        <div class="content-960 hover-btn text-center">
+            <ul>
+                <li>
+                <a href="http://www.apache.org/dyn/closer.cgi/incubator/metron/0.3.0/apache-metron-0.3.0-incubating.tar.gz" target="new">
+                    apache-metron-0.3.0-incubating.tar.gz
+                </a>
+                 [ <a href="http://www.apache.org/dyn/closer.cgi/incubator/metron/0.3.0/apache-metron-0.3.0-incubating.tar.gz.asc" target="new">
+                    PGP
+                 </a>]
+                 [ <a href="http://www.apache.org/dyn/closer.cgi/incubator/metron/0.3.0/apache-metron-0.3.0-incubating.tar.gz.sha" target="new">
+                    SHA512
+                 </a>]
+                 [ <a href="http://www.apache.org/dyn/closer.cgi/incubator/metron/0.3.0/apache-metron-0.3.0-incubating.tar.gz.md5" target="new">
+                    MD5
+                  </a>]
```

Comment:
Sorry, but as stated in the Jira:
- All links to the downloadable distribution artifacts MUST NOT reference the main Apache dist web site (dist.apache.org). Instead, they should use the standard scripting mechanisms to distribute the load between the mirror sites (see http://www.apache.org/dyn/closer.cgi/ ).
- All links to checksums, detached signatures and public keys MUST reference the main Apache dist web site (dist.apache.org), and use https (SSL).

Therefore, this section should read:
```
 +                <a href="http://www.apache.org/dyn/closer.cgi/incubator/metron/0.3.0/apache-metron-0.3.0-incubating.tar.gz" target="new">
 +                    apache-metron-0.3.0-incubating.tar.gz
 +                </a>
 +                 [ <a href="https://dist.apache.org/repos/dist/release/incubator/metron/0.3.0/apache-metron-0.3.0-incubating.tar.gz.asc" target="new">
 +                    PGP
 +                 </a>]
 +                 [ <a href="https://dist.apache.org/repos/dist/release/incubator/metron/0.3.0/apache-metron-0.3.0-incubating.tar.gz.sha" target="new">
 +                    SHA512
 +                 </a>]
 +                 [ <a href="https://dist.apache.org/repos/dist/release/incubator/metron/0.3.0/apache-metron-0.3.0-incubating.tar.gz.md5" target="new">
 +                    MD5
 +                  </a>]
```
The first item (the tarball) is correct, the other three must use https and point at the dist.apache.org site.
Thanks for doing this.